src/client/interpreter/configuration/interpreterSelector.ts

```diff
@@ -153,11 +171,30 @@ export class InterpreterSelector implements IInterpreterSelector {
         }
 
         // Ok we have multiple workspaces, get the user to pick a folder.
-        const workspaceFolder = await this.applicationShell.showWorkspaceFolderPick({
+
+        type WorkspaceSelectionQuickPickItem = QuickPickItem & { uri: Uri };
+        const quickPickItems: WorkspaceSelectionQuickPickItem[] = [
+            ...this.workspaceService.workspaceFolders.map(w => {
+                return {
+                    label: w.name,
+                    description: path.dirname(w.uri.fsPath),
+                    uri: w.uri
+                };
```

Comment:
```suggestion
                ({
                    label: w.name,
                    description: path.dirname(w.uri.fsPath),
                    uri: w.uri
                })
```
I don;t think you need a `return` there, putting it in parenthesis works. See https://stackoverflow.com/a/47755280

Reply:
Not a blocker.

Reply:
Ah, nice.